pep-0526.txt

```diff
@@ -174,38 +178,23 @@ by a different type::
   a: int
   a: str  # Static type checker will warn about this.
 
-``__annotations__`` is writable, so this is permitted::
+.. _classvar:
```

Comment:
I moved this paragraph below to lines 357-371 (in new version) right next to two other paragraphs where ``__annotations__`` is discussed.